launcher/ui/MainWindow.cpp

```diff
@@ -1605,6 +1605,9 @@ void MainWindow::setCatBackground(bool enabled)
         QDateTime xmas(QDate(now.date().year(), 12, 25), QTime(0, 0));
 
         QString cat = "default";
+        if(APPLICATION->settings()->get("CatStyle").toString() == "Floppa") {
+            cat = "floppa";
+        }
```

Comment:
I'd suggest having the CatStyle option in a variable `QString catStyleOpt = APPLICATION->settings()->get("CatStyle").toString();` and also I think you meant to do `else if` in the conditional statement that you added